scripts/install/install-helper-functions.sh

```diff
@@ -55,12 +55,15 @@ function safe_which() {
 function beginswith() { case "${1}" in "${2}"*) true;; *) false;; esac; }
 
 #-------------------------------------------------------------------------------
-# Helpers for finding the APT caching proxy.
+# Helpers for finding the IP gateway of this device.
 
-# 
-function proc_net_route_addr() {
+# Get the address of the gateway stored in /proc/net/route. Available inside
+# a docker container or in Ubuntu 18.04. When executed inside a container,
```

Comment:
"...inside an image,..." (is that right?)

Reply:
The image is the thing created by a builder and loaded into a container to execute (i.e. a filesystem snapshot). The container is where code executes (including during building, AFAICT).

Reply:
Oh, I thought these were copied into the machine and run on the image. Thanks.